docs/getting-started/architecture-and-components/overview.md

```diff
@@ -30,5 +30,7 @@ A complete Feast deployment contains the following components:
 * **Offline Store:** The offline store persists batch data that has been ingested into Feast. This data is used for producing training datasets. For feature retrieval and materialization, Feast does not manage the offline store directly, but runs queries against it. However, offline stores can be configured to support writes if Feast configures logging functionality of served features.
 
 {% hint style="info" %}
-Java and Go Clients are also available for online feature retrieval.
+Java and Go Clients are also available for online feature retrieval. 
+
+In general, we recommend [using Python](language.md) for your Feature Store microservice.
```

Comment:
The problem I see with this recommendation is online feature retrieval latency. Python has high latencies compared to Go or Java option. Do you think its better to mention the latency impact?

Reply:
I tried to address this point in this statement:
>Precomputing features is the recommended optimal path to ensure low latency performance. Reducing feature serving to a lightweight database lookup is the ideal pattern, which means the marginal overhead of Python should be tolerable. 

But I can be more explicit.

Reply:
That definitely should be a factor. Even if you precompute, there will be applications out there with low-latency requirements and high enough load for which python server performance itself might become a bottleneck. I guess we are sort of trying to address that with introducing `asyncio` in python online retrieval, but even that might not be enough for some use cases.

Reply:
While that is true in theory, in practice Python works very well at quite high scale so my goal is to make it clear that we recommend Python. 

Regardless, I see that this is in the overview section so I'll add this snippet in it.